Library/Homebrew/cask/cask_loader.rb

```diff
@@ -439,7 +439,7 @@ def self.for(ref, need_path: false, warn: true)
     end
 
     def self.default_path(token)
-      CoreCaskTap.instance.cask_dir/"#{token.to_s.downcase}.rb"
+      CoreCaskTap.instance.new_cask_path(token.to_s.downcase)
```

Comment:
Based on how this is used. I reckon this should be more like `Formulary.core_path`?

```suggestion
      find_cask_in_tap(token.to_s.downcase, CoreCaskTap.instance)
```